### ISSUE TITLE
Not loading in HA 2024.12.1

### ISSUE THREAD

**Issue by uvjim** [uvjim/linksys_velop#541]
Installed the latest HA Version core 2024.12.1.

Now I get the error: 

```
Logger: homeassistant.setup
Quelle: setup.py:334
Erstmals aufgetreten: 21:13:54 (1 Vorkommnisse)
Zuletzt protokolliert: 21:13:54

Setup failed for custom integration 'linksys_velop': Unable to import component: cannot import name 'entity_registry' from 'homeassistant.config_entries' (/usr/src/homeassistant/homeassistant/config_entries.py)
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/loader.py", line 1014, in async_get_component
    comp = await self.hass.async_add_import_executor_job(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        self._get_component, True
        ^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/usr/local/lib/python3.13/concurrent/futures/thread.py", line 58, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/usr/src/homeassistant/homeassistant/loader.py", line 1074, in _get_component
    ComponentProtocol, importlib.import_module(self.pkg_path)
                       ~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/util/loop.py", line 200, in protected_loop_func
    return func(*args, **kwargs)
  File "/usr/local/lib/python3.13/importlib/__init__.py", line 88, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1387, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1360, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1331, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 935, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 1022, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/config/custom_components/linksys_velop/__init__.py", line 10, in <module>
    from homeassistant.config_entries import entity_registry as er
ImportError: cannot import name 'entity_registry' from 'homeassistant.config_entries' (/usr/src/homeassistant/homeassistant/config_entries.py)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/setup.py", line 334, in _async_setup_component
    component = await integration.async_get_component()
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/loader.py", line 1034, in async_get_component
    self._component_future.result()
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "/usr/src/homeassistant/homeassistant/loader.py", line 1026, in async_get_component
    comp = self._get_component()
  File "/usr/src/homeassistant/homeassistant/loader.py", line 1074, in _get_component
    ComponentProtocol, importlib.import_module(self.pkg_path)
                       ~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/util/loop.py", line 200, in protected_loop_func
    return func(*args, **kwargs)
  File "/usr/local/lib/python3.13/importlib/__init__.py", line 88, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1387, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1360, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1331, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 935, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 1022, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/config/custom_components/linksys_velop/__init__.py", line 10, in <module>
    from homeassistant.config_entries import entity_registry as er
ImportError: cannot import name 'entity_registry' from 'homeassistant.config_entries' (/usr/src/homeassistant/homeassistant/config_entries.py)
```

Integration is not loaded. Using the latest release. No beta yet.

_Originally posted by @webwude in https://github.com/uvjim/linksys_velop/issues/521#issuecomment-2518483960_

**Comment by uvjim** [comment]
This was already fixed in version `2024.12.1` of this integration. I deliberately made sure that was released before the new HA version.

Can you confirm if you are using that version of the integration and still seeing this error?

**Comment by webwude** [comment]
Strange. I re-installed the already updated integration again - and it works.
